clang-tools-extra/clang-doc/assets/index.js

```diff
@@ -1,9 +1,10 @@
 function genLink(Ref) {
```

Comment:
It looks like we always invoke this function with two arguments: `Index` and `CurrentDirectory`, but we actually ignore the second argument and don't even specify it in the function signature. Do you know what is `CurrentDirectory` meant for? Could it perhaps be used instead of `Base`?

Reply:
Looks like the current directory was left around when we moved from relative paths to absolute. I don't think we can use that instead of base, though.

When I log what the CurrentDirectory is on docs.fuchsia.dev/cpp using chrome tools, it ends up empty, so I think we may still need to use Base. i'll take another look at the JS though.